config/kibana.yml

```
@@ -103,15 +103,23 @@ logging.verbose: true
 # translations.
 #i18n.defaultLocale: "en"
 
+# Reporting
+xpack.reporting.enabled: true
+xpack.reporting.encryptionKey: "fjklqwe0inqi320n1nv34u0t3"
+xpack.reporting.index: ".reporting-perchybana"
+
+xpack.reporting.csv.maxSizeBytes: 1000000000 #Default is 10MB
```

FYI: yaml supports underscores in numbers

```suggestion
xpack.reporting.csv.maxSizeBytes: 1_000_000_000  # Default is 10MB
```